src/Symfony/Component/Security/Http/EventListener/CookieClearingLogoutListener.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Http\EventListener;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Security\Http\Event\LogoutEvent;
+
+/**
+ * This listener clears the passed cookies when a user logs out.
+ *
+ * @author Johannes M. Schmitt <schmittjoh@gmail.com>
+ *
+ * @internal
+ */
+class CookieClearingLogoutListener implements EventSubscriberInterface
+{
+    private $cookies;
+
+    /**
+     * @param array $cookies An array of cookies (keys are names, values contain path and domain) to unset
+     */
+    public function __construct(array $cookies)
+    {
+        $this->cookies = $cookies;
+    }
+
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            LogoutEvent::class => ['onLogout', -255],
+        ];
+    }
+
+    public function onLogout(LogoutEvent $event): void
+    {
+        if (!$response = $event->getResponse()) {
+            return;
+        }
+
+        foreach ($this->cookies as $cookieName => $cookieData) {
+            $response->headers->clearCookie($cookieName, $cookieData['path'], $cookieData['domain']);
```

Comment:
we'd also need to pass the other attributes to properly clear any cookies, see #36173

Reply:
Hmm, interesting, seems like we forgot to fix this for the cookie clearing logout handler: https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/Security/Http/Logout/CookieClearingLogoutHandler.php

I'll make a separate PR for it.

Reply:
See #36252